aiidalab/app.py

```diff
@@ -65,6 +65,7 @@ class _AiidaLabApp:
     logo: str = field(default_factory=str)
     categories: List[str] = field(default_factory=list)
     releases: dict = field(default_factory=dict)
+    stdout = sys.stdout
```

Comment:
Why are you adding this as a class attribute?

Reply:
I answered this question in the comment below.

Reply:
responded